docs/test-fixtures.md

```diff
@@ -30,8 +47,59 @@ class UniqueTestsFixture {
  }
 ```
 
-The two test cases here will create uniquely-named derived classes of UniqueTestsFixture and thus can access the `getID()` protected method and `conn` member variables. This ensures that both the test cases are able to create a DBConnection using the same method (DRY principle) and that any ID's created are unique such that the order that tests are executed does not matter.
+The two test cases here will create uniquely-named derived classes of UniqueTestsFixture and thus can access the `getID()` protected method and `conn` member variables. This ensures that both the test cases are able to create a DBConnection using the same method (DRY principle) and that any ID's created are unique such that the order that tests are executed does not matter. 
+
+### 2. `METHOD_AS_TEST_CASE`
+
+`METHOD_AS_TEST_CASE` lets you register a member function of a class as a Catch2 test case. The class will be separately instantiated for each method registered in this way.
+
+```cpp
+class TestClass {
+    std::string s;
+
+public:
+    TestClass()
+        :s( "hello" )
+    {}
+
+    void testCase() {
+        REQUIRE( s == "hello" );
+    }
+};
+
+
+METHOD_AS_TEST_CASE( TestClass::testCase, "Use class's method as a test case", "[class]" )
+```
+
+This type of fixture is similar to [TEST_CASE_METHOD](#1-test_case_method) except in this case it will directly use the provided class to create an object rather than a derived class.
+
+### 3. `TEST_CASE_FIXTURE`
+
+> [Introduced](link-to-issue-or-PR) in Catch2 X.Y.Z
```

Comment:
I think this is the correct way to introduce a version placeholder...

Reply:
The suffix is correct (the release scripts will pick it up and update to the new version), but the target has to be done manually, so `https://github.com/catchorg/Catch2/pull/2885` instead of the `link-to-issue-or-PR`.

Reply:
Yup this was something that should have been more obvious to me... My bad. And fixed now!